.github/workflows/main.yml

```diff
@@ -0,0 +1,109 @@
+name: main
```

Comment:
I enabled this to test it out, here's a successful run:

https://github.com/ngrx/platform/actions/runs/10742124522